modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/ViewConflictsMVCRenderCommand.java

```diff
@@ -85,14 +85,27 @@ public String render(
 				HttpServletResponse httpServletResponse =
 					_portal.getHttpServletResponse(renderResponse);
 
-				httpServletResponse.sendRedirect(
-					PortletURLBuilder.createRenderURL(
-						renderResponse
-					).setMVCRenderCommandName(
-						"/change_tracking/view_changes"
-					).setParameter(
-						"ctCollectionId", ctCollectionId
-					).buildString());
+				String redirect = ParamUtil.getString(
+					renderRequest, "redirect");
+
+				if (redirect.contains("view_changes")) {
```

Comment:
Let's redirect to the given redirect if it's not null. Otherwise, manually create the redirect to the view changes page.